### PR TITLE
firestore: add LimitToLast samples

### LINKED ISSUE #1598: firestore: add LimitToLast samples
Add samples for limit to last feature, implemented in [this PR](https://github.com/googleapis/google-cloud-go/pull/2420/)

### PR DESCRIPTION
**Pull request by IlyaFaer**
Requires the [feature PR](https://github.com/googleapis/google-cloud-go/pull/2420/files) to be merged and released
Closes #1598

### CODE REVIEW

**Comment by tritone** [comment]
Looks basically fine given that the snippets in this file do not follow the canonical sample. I see that none of them are under test; do we want to add a simple test to exercise them?

Also, you'll have to pull in firestore/v1.3.0 in order for the CI to pass.

File: firestore/firestore_snippets/query.go [hunk 1]
```diff
@@ -149,6 +149,15 @@ func createOrderByNameLimitQuery(client *firestore.Client) {
 	_ = query
 }
 
+func createOrderByNameLimitToLastQuery(client *firestore.Client) {
+	cities := client.Collection("cities")
+	// [START fs_order_by_name_limit_to_last_query]
+	query := cities.OrderBy("name", firestore.Asc).LimitToLast(3)
+	// [END fs_order_by_name_limit__to_last_query]
```
Comment (BenWhitehead) [inline firestore/firestore_snippets/query.go:156]: Typo:

```suggestion
	// [END fs_order_by_name_limit_to_last_query]
```

**Comment by tritone** [comment]
Talked with @tbpg and the automated dependency update is blocked currently. @IlyaFaer could you update go.mod to use firestore/v1.3.0 and add the change to this PR?

**Comment by tbpg** [comment]
You can update the `go.mod` with:

```
go get cloud.google.com/go/firestore@v1.3.0
go mod tidy
```

**Comment by IlyaFaer** [comment]
@tritone, @tbpg, not sure how to deal with it. Sorry, I've never changed dependencies in Go manually. I've thought it's easy as renovate bot PR, but looks like I'm missing something

**Comment by IlyaFaer** [comment]
It's working fine locally (e.g. this [commit](https://github.com/GoogleCloudPlatform/golang-samples/pull/1599/commits/dfedf3b78626a3b465f6a40a2e307f1da6e6810b)), but when I'm pushing it, kokoro fails

Безымянный

**Pull request merged**